.vib/kubeapps/cypress/cypress/integration/kubeapps_spec.js

```diff
@@ -0,0 +1,30 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+
+it('allows managing a helm chart release', () => {
+  cy.login();
+
+  // Prefer UI interaction over direct navigation because the URL includes references to
+  // the ns name, which is only known at runtime.
+  cy.contains('Catalog').click();
+
+  cy.fixture('releases').then((releases) => {
+    const releaseName = `${releases.newRelease.name}-${random}`
+
+    cy.contains('div.card', releases.newRelease.application).click();
+    cy.contains('Deploy').click();
+
+    cy.get('#releaseName').type(releaseName);
+    cy.contains('Deploy').click();
+    cy.get('.application-status-pie-chart-title', { timeout: 75000 }).should(
```

Comment:
If you added this _timeout_ is because is needed but careful using harcoded values. I'd use, maybe, the one we have set in the `cypress.json` file and multiplying it for 2-3 times (to have an evidence we are using the one set as reference and avoiding "magic numbers" harcoded).

Reply:
👍 I agree we can use the `defaultCommandTimeout` as a base to derive this other timeout. Added, thanks!